pkg/virtctl/offlinevm/offlinevm.go

```diff
@@ -107,7 +109,9 @@ func (o *Command) Run(cmd *cobra.Command, args []string) error {
 
 	if ovm.Spec.Running != running {
 		ovm.Spec.Running = running
-		_, err := virtClient.OfflineVirtualMachine(namespace).Update(ovm)
+		_, err := virtClient.OfflineVirtualMachine(namespace).Patch(ovm.Name, types.MergePatchType,
+			[]byte("{\"spec\":{\"running\":true}}"))
```

Comment:
You will have to set it to false if we want to stop it.

Reply:
In general it may be beneficial now to untangle the start and stop flows in two different branches.